### PR TITLE
Add save for save next federation creation block height

### PR DESCRIPTION
**Pull request by jeremy-then**
Add save for save next federation creation block height

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java [hunk 1]
```diff
@@ -973,6 +973,32 @@ void saveActiveFederationCreationBlockHeight_whenHeightIsNotNullAndRSKIP186IsAct
 
     }
 
+    @Test
+    void saveNextFederationCreationBlockHeight_postIrisAndHeightIsNotNull_shouldSaveToStorage() {
```
Comment (nathanieliov) [inline rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java:977]:
```suggestion
    void saveNextFederationCreationBlockHeight_whenIrisIsActiveAndHeightIsNotNull_shouldSaveToStorage() {
```

**Review by nathanieliov — APPROVED** [review]
Just a small suggestion

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java (hunk 1 above)
Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java:918]: Already discussed in the poll.

**Review by wilmerrootstock — APPROVED** [review]
LGTM!

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**